README.md

```diff
@@ -4,41 +4,49 @@
 
 The above description, from the [Kubernetes homepage](https://kubernetes.io/), is centered on containerized _applications_. Yet, the Kubernetes metadata, objects, and visualizations (e.g., within Dashboard) are focused on container infrastructure rather than the applications themselves.
 
-The Application CRD [(Custom Resource Definition)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#customresourcedefinitions) and [Controller](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#custom-controllers) in this project aim to change that in a way that's interoperable between many supporting tools.
+Applications will provides a way for you to aggregate individual Kubernetes objects (e.g. Services, Deployments, StatefulSets, Ingresses, and CRDs), and manage them as a group. Hopefully providing UIs that allows for the aggregation and display of all the objects in the Application. 
```

Comment:
Replace "Applications will provides a way for you to" with
"Applications provide a way to"

Reply:
Also replace "Hopefully providing UIs that allows for the aggregation and display of all the objects in the Application."
with 
"A key goal of Applications is to enable the creation of user interfaces that allow for the aggregation and display of all the objects in the Application."